Jamulus.pro

```diff
@@ -1169,3 +1169,10 @@ contains(CONFIG, "disable_version_check") {
 
 ANDROID_ABIS = armeabi-v7a arm64-v8a x86 x86_64
 
+# Enable formatting all code via `make clang_format`.
+# Note: When extending the list of file extensions or when changing the excludes,
+# be sure to update .github/workflows/coding-style-check.yml and .clang-format-ignore as well.
+CLANG_SOURCES = $$files(*.cpp, true) $$files(*.h, true)
```

Comment:
Should we include MacOS and iOS `*.mm` files?

Reply:
In general, probably yes, but I'd rather do this (or after) the existing PR #1871 after all of the actual details have been figured out.

Reply:
Aha!